cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/labels-list.tsx

```diff
@@ -78,34 +115,42 @@ function LabelsListComponent(): JSX.Element {
+            const bothAreTags = activatedState.objectType === ObjectType.TAG && label.type === ObjectType.TAG;
+            const labelIsApplicable = label.type === LabelType.ANY ||
+                activatedState.shapeType === label.type || bothAreTags;
+            if (activatedState && labelIsApplicable) {
+                activatedState.label = label;
+                dispatch(updateAnnotationsAsync([activatedState]));
+            }
+        } else {
+            const label = labels.filter((_label: any) => _label.id === labelID)[0];
+            const bothAreTags = activeObjectType === ObjectType.TAG && label.type === ObjectType.TAG;
+            const labelIsApplicable = label.type === LabelType.ANY ||
+                activeShapeType === label.type || bothAreTags;
+            if (labelIsApplicable) {
+                dispatch(rememberObject({ activeLabelID: labelID }));
+                message.destroy();
+                message.success(`Default label has been changed to "${label.name}"`);
+            }
+        }
+    };
+
+    const handlers: Record<keyof typeof componentShortcuts, (event: KeyboardEvent, shortcut: string) => void> = {};
+
+    for (const i of [1, 2, 3, 4, 5, 6, 7, 8, 9, 0]) {
+        handlers[`SWITCH_LABEL_${i}`] = (event: KeyboardEvent) => {
             if (event) event.preventDefault();
-            const labelID = keyToLabelMapping[shortcut.split('+')[1].trim()];
+            const labelID = keyToLabelMapping[i];
             const label = labels.filter((_label: any) => _label.id === labelID)[0];
             if (Number.isInteger(labelID) && label) {
-                if (Number.isInteger(activatedStateID)) {
-                    const activatedState = states.filter((state: any) => state.clientID === activatedStateID)[0];
-                    const bothAreTags = activatedState.objectType === ObjectType.TAG && label.type === ObjectType.TAG;
-                    const labelIsApplicable = label.type === LabelType.ANY ||
-                        activatedState.shapeType === label.type || bothAreTags;
-                    if (activatedState && labelIsApplicable) {
-                        activatedState.label = label;
-                        dispatch(updateAnnotationsAsync([activatedState]));
-                    }
-                } else {
-                    const bothAreTags = activeObjectType === ObjectType.TAG && label.type === ObjectType.TAG;
-                    const labelIsApplicable = label.type === LabelType.ANY ||
-                        activeShapeType === label.type || bothAreTags;
-                    if (labelIsApplicable) {
-                        dispatch(rememberObject({ activeLabelID: labelID }));
-                        message.destroy();
-                        message.success(`Default label has been changed to "${label.name}"`);
-                    }
-                }
+                handleHelper(labelID);
```

Comment:
If you moved code to the function ``handleHelper``, is there any reason to left this line of code?
Why not put everything to the function ``handleHelper``? Currently in the function ``handleHelper`` we are looking for the label again

```
const label = labels.filter((_label: any) => _label.id === labelID)[0];
```